File: jp-livedata-java/src/main/java/com/mao/jp_livedata_java/Logger.java
package com.mao.jp_livedata_java;

import android.util.Log;


/**
 * @author zhangkun
 * @time 2022/2/16 17:23
 * @Description  日志工具类
 */
public class Logger {
    private final static boolean DEBUG = true;

    private final static String TAG = "LiveDataBus";

    public static void d(String tag, String msg){
        if (DEBUG){
            Log.d(TAG, tag + "——" +msg);
        }
    }

    public static void v(String tag, String msg){
        if (DEBUG){
            Log.v(TAG, tag + "——" +msg);
        }
    }

    public static void i(String tag, String msg){
        if (DEBUG){
            Log.i(TAG, tag + "——" +msg);
        }
    }

    public static void w(String tag, String msg){
        if (DEBUG){
            Log.w(TAG, tag + "——" +msg);
        }
    }

    public static void e(String tag, String msg){
        if (DEBUG){
            Log.e(TAG, tag + "——" +msg);
        }
    }

}

File: jp-livedata/src/main/java/com/mao/jp_livedata/Logger.kt
package com.mao.jp_livedata

import android.util.Log

/**
 * 日志工具类
 * @author linyaotian
 */
object Logger {
    private const val DEBUG = true
    private const val TAG = "LiveDataBus"
    fun d(tag: String, msg: String) {
        if (DEBUG) {
            Log.d(TAG, "$tag——$msg")
        }
    }

    fun v(tag: String, msg: String) {
        if (DEBUG) {
            Log.v(TAG, "$tag——$msg")
        }
    }

    fun i(tag: String, msg: String) {
        if (DEBUG) {
            Log.i(TAG, "$tag——$msg")
        }
    }

    fun w(tag: String, msg: String) {
        if (DEBUG) {
            Log.w(TAG, "$tag——$msg")
        }
    }

    fun e(tag: String, msg: String) {
        if (DEBUG) {
            Log.e(TAG, "$tag——$msg")
        }
    }
}